src/components/UI/Input/InputTypeThree.jsx

```diff
@@ -0,0 +1,30 @@
+import "./InputTypeThree.css";
+export const InputTypeThree = (props) => {
+	const {
+		wrapperClassName,
+		htmlFor,
+		labelClassName,
+		labelText,
+		type,
+		className,
+		placeholder,
+		name,
+		iconClassName,
+	} = props;
+	return (
+		<div className={wrapperClassName}>
+			<label htmlFor={htmlFor} className={labelClassName}>
+				{labelText}
+			</label>
+			<input
+				type={type}
+				className={className}
+				placeholder={placeholder}
+				required
+			/>
```

Comment:
You can do this code as the below one.

<div class="wrapper">
<Input />
<Button>
</>

Reply:
Got your point. Will update it